### ISSUE TITLE
tempfile not available on non-debian-based linux distributions

### ISSUE THREAD

**Issue by alainsanguinetti** [RethinkRobotics/baxter#171]
Hi, I just tried to run the script on arch linux and it failed saying that tempfile is not available.

I found a working solution : 

# https://github.com/nilnor/howl/commit/93abf093af6fd0c4d5adfbdffe49ed64c3be36a2
```bash
if [ -z "$tmp" ]; then
	tmpdir=$TMPDIR
	if [ -z "$tmpdir" ]; then
		tmpdir=/tmp
	fi
	tmp=$tmpdir/howl_download$RANDOM.$$
fi
tf=$tmp 
```

**Comment by rethink-imcmahon** [comment]
Interesting! Thank you for reporting this issue along with your solution. I will test it out and merge into the Development branch :)

**Comment by rethink-imcmahon** [comment]
After talking with one of our Linux guru's, the command `mktemp` was suggested as an alternative to `tmpdir` since it is part of the `coreutils` package. Does simply replacing `tmpdir` with this work for you? Also, what Linux distro are you using, so I can replicate the issue?

**Comment by alainsanguinetti** [comment]
mktemp works very well for me, much more elegant solution than the above, thank you very much !
I am using arch linux.

**Comment by rethink-imcmahon** [comment]
Wonderful! I have added this fix into the release-1.1.1 patch. I'll create a pull request momentarily. Even though this is a simple change, I'd love for you to look it over and give a +1 @alainsanguinetti
